pydoctor/sphinx.py

```diff
@@ -109,14 +109,16 @@ def _parseInventory(
         """
         result = {}
         for line in payload.splitlines():
-            parts = line.split(' ', 4)
-            if len(parts) != 5:
+            try:
+                name, typ, prio, location, display = _parseInventoryLine(line)
+            except ValueError:
                 self.error(
                     'sphinx',
                     'Failed to parse line "%s" for %s' % (line, base_url),
                     )
                 continue
-            result[parts[0]] = (base_url, parts[3])
+            if typ.startswith('py:'):
+                result[name] = (base_url, location)
```

Comment:
```suggestion
            if not typ.startswith('py:'):
                # Non python references are ignored.
                continue
            result[name] = (base_url, location)
```

Reply:
We already have a `continue` statement in the method, so I guess this is consistent.

Reply:
this is a minor comment.

is a question of  style - https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

the idea is that the happy path handling is non indented and not "hidden" in a conditional branch.

Reply:
I'm not sure this should be considered a happy path, in the sense that there is nothing wrong with non-Python references, but we're currently just not interested in them.